### PR TITLE
Abb 48 nv

### PR DESCRIPTION
**Pull request by nafisa-valieva**
I have started to work on Abb 48, it is not over yet, but it will take some time. I want to merge it now because it is referred to in other files.

### CODE REVIEW

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: ParisBNF/abb/BNFabb48.xml [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+              <msIdentifier>
+                  <repository ref="INS0303BNF"/>
+                  <collection>Manuscrits orientaux</collection>
+                  <collection>Fonds éthiopien</collection>
+                  <collection>d'Abbadie</collection>
+                 <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b100877006#"> BnF Éthiopien d'Abbadie 48</idno>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb48.xml:30]: remove the `# `at the end

File: ParisBNF/abb/BNFabb48.xml [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+                    </msIdentifier>
+               <msContents>
+                  <summary>
+                  </summary>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb48.xml:34]: you can sometimes use the "Format and Indent" button 
image


 to have the XML look clearer and all nestings indented properly

File: ParisBNF/abb/BNFabb48.xml [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+                     </list>
+                    <list>
+                      <item xml:id="a1">
+                        <locus target="#1r"/>
+                         <desc type="Unclear"/>
+                         <p>ዋ፡ ዛፍ፡ በሀ፡ ያቤ፡ አደበጃን፡ ፩ጋ፡ ሸዌ፡ ዕርገቱ፡</p>
+                      </item>
+                       <item xml:id="a2">
+                          <locus target="#2r"/>
+                          <title type="incomlete" ref="LIT3327Melchiz2"/>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb48.xml:92]: `incomplete`

careful (I do not know why the error is not signalled)

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Review by karljonaskarlsson — APPROVED** [review]
I leave comments on minor details to when the full description is pull-requested :)

File: ParisBNF/abb/BNFabb48.xml (hunk 1 above)
Comment (nafisa-valieva) [reply ParisBNF/abb/BNFabb48.xml:30]: done!

File: ParisBNF/abb/BNFabb48.xml (hunk 2 above)
Comment (nafisa-valieva) [reply ParisBNF/abb/BNFabb48.xml:34]: Ok! I need to learn how to use it.

File: ParisBNF/abb/BNFabb48.xml (hunk 3 above)
Comment (nafisa-valieva) [reply ParisBNF/abb/BNFabb48.xml:92]: corrected, thank you!

**Comment by nafisa-valieva** [comment]
> I leave comments on minor details to when the full description is pull-requested :)

yes, this is the right way :-)

File: ParisBNF/abb/BNFabb48.xml (hunk 2 above)
Comment (nafisa-valieva) [reply ParisBNF/abb/BNFabb48.xml]: done!

File: ParisBNF/abb/BNFabb48.xml [hunk 4]
```diff
@@ -22,56 +23,56 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
             <msDesc xml:id="ms">
-              <msIdentifier>
+               <msIdentifier>
                   <repository ref="INS0303BNF"/>
                   <collection>Manuscrits orientaux</collection>
                   <collection>Fonds éthiopien</collection>
                   <collection>d'Abbadie</collection>
-                 <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b100877006#"> BnF Éthiopien d'Abbadie 48</idno>
-                    </msIdentifier>
+                  <idno facs="https://gallica.bnf.fr/ark:/12148/btv1b100877006#"> BnF Éthiopien
```
[the remark this answers is not in the document]
Comment (nafisa-valieva) [reply ParisBNF/abb/BNFabb48.xml]: done

**Review by eu-genia — APPROVED** [review]

**Comment by eu-genia** [comment]
for the future please make sure that there is no `p` in `license`

**Pull request merged**